api/src/main/java/org/hawkular/apm/api/utils/PropertyUtil.java

```diff
@@ -111,6 +111,11 @@
     public static final String HAWKULAR_APM_COLLECTOR_BATCHTHREADS = "HAWKULAR_APM_COLLECTOR_BATCHTHREADS";
 
     /**
+     * The time interval (in milliseconds) between checks for expired trace states in the OpenTracing agent.
+     */
+    public static final String HAWKULAR_APM_OTAGENT_EXPIRY_INTERVAL = "HAWKULAR_APM_OTAGENT_EXPIRY_INTERVAL";
```

Comment:
Updated to HAWKULAR_APM_AGENT_STATE_EXPIRY_INTERVAL - which better reflects what it represents.